var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -102,6 +102,7 @@ def url_for_version(self, version):
 
     # High-level API of HDF5 1.8.9 or later is required for netCDF-4 support:
     # http://www.unidata.ucar.edu/software/netcdf/docs/getting_and_building_netcdf.html
+    depends_on('hdf5@1.8.9:+hl api=v18', when='@4.8.0:')
```

Comment:
According to the PR description, shouldn't this read:
```suggestion
    depends_on('hdf5@1.12.0:+hl api=v18', when='@4.8.0:')
```
?